Add FAQ Section (#322)
helium/docs: docs/faq/helium-network.mdx

---
id: helium-network 
hide_title: true
title: Helium Network FAQ
sidebar_label: Helium Network
slug: /faq/helium-network
---

# FAQ - Helium Network

### Does this replace the Wi-Fi in my house?

> No, the Helium Hotspot uses Long-Fi broadcasts a radio signal, connecting low-power devices in your area that need to transmit data....devices such as GPS trackers, environmental sensors etc. that only need to transmit and share small bits of information (like a location or temperature). It uses your existing internet to connect to peers over on the Helium network and deliver the data packets sent by devices.  It does not replace internet or cellular service for regular devices like computers and smartphones.

Reviewer: ```suggestion
No. The Helium Hotspot creates a Long-Fi radio signal and routes packets from low-power devices in your area that use LoRaWAN and have been deployed to the Helium Network. Typically these are devices such as GPS trackers, environmental sensors, weather meters, etc., that only need to transmit and share small bits of information. The Hotspot uses your existing Internet (via WiFi or Ethernet) to deliver the data packets sent by devices.  It does not replace internet or cellular service for regular devices like computers and smartphones.
```

 
### Can I connect my phone or computer to Hotspots?

> No you can't. The network is meant for transferring small amounts of data at this time.  

Reviewer: ```suggestion
No. The network is only meant devices that use LoRaWAN.
```

  
### What devices can use the network?

> Everything from asset trackers, to soil sensors in farms, to tracking the water level of bottles in water coolers. Anyone can use LongFi, which works on existing IoT hardware, to create lots of different products. 

Reviewer: ```suggestion
Anything that sends data via LoRaWAN. This means everything from asset trackers, to soil sensors in farms, to tracking the water level of bottles in water coolers. [There is an extensive, partial list here](/use-the-network/devices). 
```

  
### What is a Proof-of-Coverage (PoC) Challenge?

> A Challenge requires a small group of hotspots to prove that they are in the location that they say they are.  If they truly are there and online, and within radio distance of each other, then they can hear the "call" and respond as a witness to validate their position.  For a deeper breakdown click here.

Reviewer: ```suggestion
A Challenge requires a small group of hotspots to prove that they are in the location that they say they are.  If they truly are there and online, and within radio distance of each other, then they can hear the "call" and respond as a witness to validate their position.  For a deeper breakdown [click here](/blockchain/proof-of-coverage/#proof-of-coverage).
```

### What if I'm in a rural area, or the only hotspot in my area?  Can I still earn rewards?
Hotspots earn by providing and validating network coverage provided by other hotspots. This includes both sending a challenge over the internet or validating coverage of other hotspots within range. 
 

> It's best to have at least 1 or 2 other hotspots within radio range of you, so that you can participate in PoC Challenges to earn rewards.  If you are the only hotspot in your area, you can still earn tokens if a device uses your signal and by initiating the Challenges, though.  Tokens are distributed based on these economics.

Reviewer: ```suggestion
It's best to have at least one or two other hotspots within radio range of you so that you can participate in PoC Challenges to earn rewards a higher rate of HNT.  If you are the only Hotspot in your area, you can still earn HNT if a device uses your Hotspots and by initiating the Challenges.  Tokens are distributed based on these economics.
```
```suggestion
It's best to have at least 1 or 2 other hotspots within radio range of you, so that you can participate in PoC Challenges to earn rewards.  If you are the only hotspot in your area, you can still earn tokens if a device uses your signal and by initiating the Challenges, though.  [Tokens are distributed based on these economics](/blockchain/mining).
```

> ### Can I have 2 or 3 hotspots at my house?

Reviewer: ```suggestion
### Can I have more than one Hotspot at the same location?
```

> If hotspots are too close (within 300-350 meters or so) they will end up not being able to witness each other and may compete with each other for Challenges. Their overall participation in Proof-of-Coverage would be impacted also, since rewards are also density based.  It's better to have them spaced out, at least a few houses or blocks away, to broaden network coverage.  

Reviewer: ```suggestion
If Hotspots are too close - less than 300 meters - they will not be eligible twitness each other as part of POC challenges and may compete with each other for challenges. Their overall participation in Proof-of-Coverage would be impacted also, since rewards are also density based.  Proof of coverage rewards good, wide coverage. You'll earn more HNT if you do this. 
```

 
### How do I earn Helium?

> Once set up and connected to the internet, there are 5 ways your hotspot earns (with no additional work from you!):

Reviewer: ```suggestion
Once deployed via the Helium Wallet and fully synced, there are five ways your Hotspot earns HNT.  A full list of [mining rewards can be found here](/blockchain/mining). 
```

> - When a device sends information over the network using your hotspot

Reviewer: ```suggestion
```

> - By initiating PoC Challenges over the internet

Reviewer: ```suggestion
```

> - By participating in a PoC Challenge

Reviewer: ```suggestion
```

> - By acting as a Witness to a PoC Challenge between nearby hotspots.

Reviewer: ```suggestion
```

> - Being a part of a Consensus Group

Reviewer: ```suggestion
```

> The Tokens Page can elaborate a bit more if needed.  

Reviewer: ```suggestion
```

  
### How much Helium can I earn?

> It's hard to say since there are so many variables that go into it, i.e. the total number of active hotspots, how many hotspots are within range of yours, how many devices use your hotspot, etc.  That said, the sooner you join, the sooner you start mining Helium tokens!  Note that we are only able to talk about potential earnings in regards to Helium tokens and the network.

Reviewer: ```suggestion
Hotspot earnings are imprecise and hard to predict for many reasons. Primarily this is due to the number of variables that account for HNT earnings at any given time. Do your own research and work hard to optimize your deployments. Better coverage yields more HNT.

```

 
### Is there a limit to the number of hotspots per city?

> At this time there is no hard cap on the number of Hotspots per city. In fact, we believe more coverage per city will encourage more usage.  
>  

Reviewer: ```suggestion
There is no hard cap on the number of Hotspots per city. In fact, we believe more coverage per city will encourage more usage.  You can see a full city leaderboard [on the Helium Explorer](https://explorer.helium.com/hotspots).
 
```
